php/bootstrap.php

```diff
@@ -19,9 +19,9 @@ function get_bootstrap_steps() {
 		'EE\Bootstrap\DeclareMainClass',
 		'EE\Bootstrap\DeclareAbstractBaseCommand',
 		'EE\Bootstrap\IncludeFrameworkAutoloader',
-		'EE\Bootstrap\ConfigureRunner',
 		'EE\Bootstrap\InitializeColorization',
 		'EE\Bootstrap\InitializeLogger',
+		'EE\Bootstrap\ConfigureRunner',
```

Comment:
What's the reason behind changing the order?
What would happen if we don't change it?

Reply:
@kirtangajjar I guess it is to update the loading sequence to show argument errors in Runner. https://github.com/EasyEngine/easyengine/pull/1050/commits/da0fcf9db50474a2d24d5ed6f33a63b565e09955